packages/avnu_provider/pubspec.yaml

```
@@ -0,0 +1,22 @@
+  starknet: ^0.1.0+1
+  freezed_annotation: ^2.4.4
+  http: ^1.2.2
+  json_annotation: ^4.9.0
+  pointycastle: ^3.9.1
+
+dev_dependencies:
+  build_runner: ^2.4.13
+  freezed: ^2.5.7
+  json_serializable: ^6.8.0
+  lints: ^5.0.0
+  test: ^1.25.8
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)